libraries/SRV_Channel/SRV_Channels.cpp

```diff
@@ -221,6 +230,17 @@ void SRV_Channels::set_output_pwm_chan(uint8_t chan, uint16_t value)
     }
 }
 
+// set output value for a specific function channel as a pwm value with loop based timeout
+void SRV_Channels::set_output_pwm_chan_counter(uint8_t chan, uint16_t value, uint16_t counter)
+{
+    WITH_SEMAPHORE(_singleton->calc_sem);
+
+    if (chan < NUM_SERVO_CHANNELS) {
+        override_counter[chan] = counter;
+    }
+    SRV_Channels::set_output_pwm_chan(chan, value);
```

Comment:
Why call this if you know the chan is out of range? Why not move this into the `if`?

Reply:
Initially writing it I was trying to mimic the other functions nearby. Reading it that makes more sense. Changed and pushed.